Setting quarkus.resteasy.path not reflected in OpenAPI or Swagger

### Describe the bug

When I set the "quarkus.resteast.path" it works making API calls directly but does not get reflected in the OpenAPI or Swagger.

Using @ApplicationPath and setting eash @Path looks to work.

### Expected behavior

OpenAPI and Swagger reflect the path set in the application properties.

### Actual behavior

RESTful calls works but OpenAPI and Swagger do not reflect the path.  From Swagger invoking an endpoint gives a 404.

### How to Reproduce?

_No response_

### Output of `uname -a` or `ver`

_No response_

### Output of `java -version`

_No response_

### GraalVM version (if different from Java)

_No response_

### Quarkus version or git rev

_No response_

### Build tool (ie. output of `mvnw --version` or `gradlew --version`)

_No response_

### Additional information

_No response_

The "should_have_v1" test give me the error using either the yaml/properties file.  The "should_not_have_v1" passes.

[ERROR] Failures:
[ERROR]   OpenApiTest.should_have_v1_in_path:19 1 expectation failed.
JSON path paths doesn't match.
Expected: map containing ["/v1/ping/me"->ANYTHING]
  Actual: <{/ping/me={get={responses={200={description=OK, content={application/json={schema={type=boolean}}}}}}}}>

[ERROR]   ResourceTest.should_ping_api_using_path:32 1 expectation failed.
Expected status code <200> but was <404>.

[ERROR]   ResourceTest.should_ping_api_without_path:24 1 expectation failed.
Expected status code <404> but was <405>.

```java
@QuarkusTest
public class OpenApiTest {
        private static final String OPEN_API_PATH = "/q/openapi";

        @Test
        public void should_have_v1_in_path() {
                RestAssured.given().queryParam("format", "JSON").when().get(OPEN_API_PATH).then()
                                .header("Content-Type", "application/json;charset=UTF-8")
                                .body("openapi", Matchers.startsWith("3.0"))
                                .body("info.title", Matchers.equalTo("Generated API"))
                                .body("paths", Matchers.hasKey("/v1/ping/me"));
        }

        @Test
        public void should_not_have_v1_in_path() {
                RestAssured.given().queryParam("format", "JSON").when().get(OPEN_API_PATH).then()
                                .header("Content-Type", "application/json;charset=UTF-8")
                                .body("openapi", Matchers.startsWith("3.0"))
                                .body("info.title", Matchers.equalTo("Generated API"))
                                .body("paths", Matchers.hasKey("/ping/me"));
        }
}
```

--- a/extensions/smallrye-openapi/deployment/src/main/java/io/quarkus/smallrye/openapi/deployment/SmallRyeOpenApiProcessor.java
+++ b/extensions/smallrye-openapi/deployment/src/main/java/io/quarkus/smallrye/openapi/deployment/SmallRyeOpenApiProcessor.java
@@ -496,17 +496,22 @@ public class SmallRyeOpenApiProcessor {
         OpenApiConfig openApiConfig = new OpenApiConfigImpl(config);
 
         List<AnnotationScannerExtension> extensions = new ArrayList<>();
+
         // Add the RESTEasy extension if the capability is present
+        String defaultPath = httpRootPathBuildItem.getRootPath();
         if (capabilities.isPresent(Capability.RESTEASY)) {
             extensions.add(new RESTEasyExtension(indexView));
+            if (resteasyJaxrsConfig.isPresent()) {
+                defaultPath = resteasyJaxrsConfig.get().getRootPath();
+            }
+        } else if (capabilities.isPresent(Capability.RESTEASY_REACTIVE)) {
+            extensions.add(new RESTEasyExtension(indexView));
+            Optional<String> maybePath = config.getOptionalValue("quarkus.resteasy-reactive.path", String.class);
+            if (maybePath.isPresent()) {
+                defaultPath = maybePath.get();
+            }
         }
 
-        String defaultPath;
-        if (resteasyJaxrsConfig.isPresent()) {
-            defaultPath = resteasyJaxrsConfig.get().getRootPath();
-        } else {
-            defaultPath = httpRootPathBuildItem.getRootPath();
-        }
         if (defaultPath != null && !"/".equals(defaultPath)) {
             extensions.add(new CustomPathExtension(defaultPath));
         }

